Lib/unittest/test/testmock/testasync.py

```diff
@@ -138,14 +138,43 @@ def foo(): pass
         self.assertTrue(inspect.iscoroutinefunction(mock))
 
     def test_future_isfuture(self):
-        loop = asyncio.new_event_loop()
-        asyncio.set_event_loop(loop)
         fut = asyncio.Future()
-        loop.stop()
-        loop.close()
         mock = AsyncMock(fut)
         self.assertIsInstance(mock, asyncio.Future)
 
+    async def test_return_value_awaitable(self):
```

Comment:
Requested offline some tests that will cover the functionality of setting AsyncMock's as the return_value and side_effects of AsyncMock's.